src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.html

```diff
@@ -0,0 +1,6 @@
+<h2>{{header}}</h2>
+<p>{{text}}</p>
+
+<button class="btn btn-primary" (click)="close()">
```

Comment:
The button is missing the mat-button attribute.

Reply:
added :)